### ISSUE TITLE
Brighter console prompt `>`

### ISSUE THREAD

**Issue by rjake** [posit-dev/positron#3970]
In all of the dark theme options, the prompt is very hard to see. Is there a way to make the `>` line brighter? I looked through the `console.` setting options and haven't found anything that updates this. I know the 🟥 indicates that my code is still running but I'm so used to looking for the `>` to reappear.

![image](https://github.com/posit-dev/positron/assets/14003618/e1ef43e0-ad33-4e3d-948f-759e89d61462)

**Comment by LGraz** [comment]
In vscode/terminal I have a visible pink prompt via the `.Rprofile`:
`options(prompt = "\001\033[0;35m\002>\001\033[0m\002 ")`

maybe this helps :)

**Comment by rjake** [comment]
@LGraz no dice
![image](https://github.com/posit-dev/positron/assets/14003618/049b9dbe-2f78-458c-8b7f-cc568d105d39)

**Comment by juliasilge** [comment]
Can you tell me specifically tell me which theme you are using? And are you saying the active prompt is _always_ dim, even when the console is idle? Or only while code is executing?

**Comment by rmflight** [comment]
I'm using Positron Dark. And the active prompt is dim when nothing is being typed. I agree, it's hard to tell that the console is ready to use again.

In this image, where I wrote some code, the prompt is bright, and was bright as I was typing. But then when it's "waiting" again, it's dim, and almost impossible to see.

![image](https://github.com/user-attachments/assets/e4e7e4c7-fcf0-4722-b167-a885493d8dfa)

**Comment by juliasilge** [comment]
OK, turns out this a Linux-only problem with the console! 

![image](https://github.com/user-attachments/assets/f2df95e3-7b50-484b-aac5-7b9560a200c6)

Thanks so much to both of you for the report.

**Comment by rjake** [comment]
@juliasilge I'm using Windows 10 and seeing this behavior. Am I misunderstanding your note about Linux-only?

**Comment by juliasilge** [comment]
Oh hmmmm, nope, you do not misunderstand. We were only able to reproduce it on Linux internally. I'll remove the Linux tag and we'll see how things turn out as we solve the problem.

**Comment by JeffreyRStevens** [comment]
I've noticed this issue in all 10 of the dark themes that I've tried (in Ubuntu). I'm messing around with some theming for other colors. Can the console prompt color be controlled in a theme? If so, what is the theme component to control it?

**Comment by juliasilge** [comment]
I don't think so, unfortunately @JeffreyRStevens. I don't think there is a workaround and we just need to fix this bug. 😬

**Comment by JeffreyRStevens** [comment]
OK, thanks, @juliasilge!

**Comment by juliasilge** [comment]
Reported again at https://github.com/posit-dev/positron/discussions/4443
